pkgs/desktops/plasma-5/kwin/default.nix

```diff
@@ -21,22 +21,21 @@ mkDerivation {
     epoxy libICE libSM libinput libxkbcommon udev wayland xcb-util-cursor
     xwayland
 
-    qtdeclarative qtmultimedia qtscript qtx11extras
+    qtdeclarative qtmultimedia qtscript qtx11extras qtsensors
 
     breeze-qt5 kactivities kcmutils kcompletion kconfig kconfigwidgets
     kcoreaddons kcrash kdeclarative kdecoration kglobalaccel ki18n kiconthemes
     kidletime kinit kio knewstuff knotifications kpackage kscreenlocker kservice
     kwayland kwidgetsaddons kwindowsystem kxmlgui plasma-framework
+    libcap
   ];
   outputs = [ "bin" "dev" "out" ];
-  patches = copyPathsToStore (lib.readPathsFromFile ./. ./series)
-    ++ [(fetchpatch {
-        name = "cmake-3.10.diff";
-        # included upstream for kwin >= 5.11.95
-        url = "https://github.com/KDE/kwin/commit/cd544890ced4192.diff";
-        sha256 = "0z5nbcg712v10mskb7r9v0jcx5h8q4ixb7fjbb0kicmzsc266yd5";
-    })]
-    ;
+  patches = copyPathsToStore (lib.readPathsFromFile ./. ./series) ++ [
```

Comment:
Please add in a comment mentioning when we can drop this patch again.

Reply:
Done